ui/src/LandingPage.js

```diff
@@ -1,104 +1,39 @@
 import React, { Component } from "react";
 import "@fontsource/roboto";
-import {
-  Button,
-  Dialog,
-  DialogContent,
-  TextField,
-  Paper,
-  FormControl,
-  InputLabel,
-  Select,
-  MenuItem,
-  Typography,
-} from "@material-ui/core";
-import Table from "@material-ui/core/Table";
-import TableBody from "@material-ui/core/TableBody";
-import TableCell from "@material-ui/core/TableCell";
-import TableContainer from "@material-ui/core/TableContainer";
-import TableHead from "@material-ui/core/TableHead";
-import TableRow from "@material-ui/core/TableRow";
 import { Link } from "react-router-dom";
+import Surveys from "./SurveysList";
+import ExportFileTemplate from "./ExportFileTemplatesList";
+import SmsTemplatesList from "./SmsTemplatesList";
+import EmailTemplateList from "./EmailTemplateList";
+import ConfigureFulfilmentTrigger from "./ConfigureFulfilmentTrigger";
+
+import { getAuthorisedActivities } from "./Utils";
 
 class LandingPage extends Component {
   state = {
     authorisedActivities: [],
     thisUserAdminGroups: [],
-    surveys: [],
-    createSurveyDialogDisplayed: false,
-    validationError: false,
-    newSurveyName: "",
-    validationRulesValidationError: false,
-    sampleDefinitionUrlError: false,
-    surveyMetadataError: false,
-    newSurveyValidationRules: "",
-    newSurveySampleDefintionUrl: "",
-    newSurveyMetadata: "",
-    exportFileTemplates: [],
-    smsTemplates: [],
-    emailTemplates: [],
-    createExportFileTemplatePackCodeError: "",
-    createExportFileTemplateDialogDisplayed: false,
-    createSmsTemplateDialogDisplayed: false,
-    createSmsTemplateError: "",
-    createSMSTemplatePackCodeError: "",
-    createEmailTemplateDialogDisplayed: false,
-    createEmailTemplateError: "",
-    createEmailTemplatePackCodeError: "",
-    exportFileDestinations: [],
-    exportFileDestination: "",
-    description: "",
-    packCode: "",
-    template: "",
-    newTemplateMetadata: "",
-    notifyTemplateId: "",
-    exportFileDestinationValidationError: false,
-    descriptionValidationError: false,
-    packCodeValidationError: false,
-    templateValidationError: false,
-    newTemplateMetadataValidationError: false,
-    notifyTemplateIdValidationError: false,
-    notifyTemplateIdErrorMessage: "",
-    newSurveyHeaderRow: true,
-    newSurveySampleSeparator: ",",
-    nextFulfilmentTriggerDateTime: "1970-01-01T00:00:00.000Z",
-    configureNextTriggerDisplayed: false,
   };
 
   componentDidMount() {
     this.getAuthorisedBackendData();
   }
 
-  componentWillUnmount() {
-    clearInterval(this.interval);
-  }
+  // componentWillUnmount() {
```

Comment:
Think this is still needed and shouldn't be commented out?

Reply:
No use of interval on this page anymore